.scalafmt.conf

```
@@ -0,0 +1,12 @@
+version=2.6.4
+project.git=true
+style = defaultWithAlign
+maxColumn=120
+align = more
+align.tokens.add = [ ":", ":=", "=" ]
+importSelectors = singleLine
+trailingCommas = never
+verticalMultiline.atDefnSite = true
+verticalMultiline.arityThreshold = 4
+verticalMultiline.newlineAfterOpenParen = true
+verticalMultiline.newlineBeforeImplicitKW = true
```

We have this conf in frontend that we think should work nicely :)

```suggestion
version = 3.6.1
runner.dialect = scala213
lineEndings = unix
importSelectors = singleLine
maxColumn = 100

project {
  git = true
}

align {
  preset = most
  tokens = [ {code = "=>", owner = "Case|Type.Arg.ByName"}, "=", "<-", "->", "%", "%%", "should", "shouldBe", "shouldEqual", "shouldNot", "must" ]
  arrowEnumeratorGenerator = true
  openParenCallSite = false
  openParenDefnSite = false
}

continuationIndent {
  callSite = 2
  defnSite = 2
}


spaces {
  inImportCurlyBraces = false
  beforeContextBoundColon = Never
}

newlines {
  beforeCurlyLambdaParams = multiline
  afterCurlyLambdaParams = squash
}

rewrite {
  rules = [RedundantBraces, Imports],
  imports.sort = scalastyle,
  imports.groups = [
    ["java\\..*", "javax\\..*"],
    ["scala\\..*"],
    ["play\\..*", "uk.gov.hmrc\\..*"],
    ["cats\\..*"],
    [".*"]
  ]
}
```